lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,31 @@
+
+{{alias}}( re1, im1, re2, im2 )
```

Comment:
You may have copied this from `multiply`. In which case, this file needs to be updated.